releasenotes/notes/add-tracer-flare-65e275bca27631dd.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    tracer: This introduces the tracer flare functionality. Currently the tracer flare includes the
```

Comment:
do we have any public docs we can link to on what this is/how to use it?

Reply:
As far as I'm aware, no 😢 I think now that the other tracers are implementing the flare, there's been some talk about having a centralized doc for this